modules/grafana/templates/dashboards/deployment_panels/_links.json.erb

```diff
@@ -1,5 +1,10 @@
 {
-  "content": "<ul style=\"font-size: 20px;\">\n<li><a target=\"_blank\" href=\"https://errbit.<%= @app_domain %>/\">Errbit</a></li>\n</ul>\n",
+  "content": "<ul style=\"font-size: 20px;\">\n
+      <li><a target=\"_blank\" href=\"https://errbit.<%= @app_domain %>/\">Errbit</a></li>\n
+      <li><a target=\"_blank\" href=\"https://alert.<%= @app_domain %>/\">Icinga</a></li>\n
+      <li><a target=\"_blank\" href=\"https://docs.publishing.service.gov.uk/apps/<%= @app_name %>.html\"><%= @app_name %> documentation</a></li>\n
+      <li><a target=\"_blank\" href=\"https://govuk.zendesk.com/agent/filters/135081425\">Feedback</a></li>\n
```

Comment:
Did we want to add a github link or are we assuming they can get there themselves?